cmd/commands/runtime_install.go

```diff
@@ -261,7 +269,7 @@ func runtimeInstallCommandPreRunHandler(cmd *cobra.Command, opts *RuntimeInstall
 		return err
 	}
 
-	err = ensureIngressClass(ctx, opts)
+	err = ensureControllerSupported(ctx, opts)
```

Comment:
I don't think `controller` is a good name, it's not clear which controller. I prefer either leave it as `ingressController` or `routingController`.